src/main/java/seedu/edulink/logic/parser/AddCommandParser.java

```diff
@@ -41,12 +41,14 @@ public AddCommand parse(String args) throws ParseException {
             ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL,
                 PREFIX_ADDRESS, PREFIX_TAG, PREFIX_ID, PREFIX_MAJOR, PREFIX_INTAKE);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_ID, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL)
+        if (!arePrefixesPresent(argMultimap, PREFIX_ID, PREFIX_NAME, PREFIX_ADDRESS,
```

Comment:
Nice to make them compulsory now. Given our future iterations look to perform calculations on these values.